### ISSUE TITLE
Results for VisDA-C dadaset

### ISSUE THREAD

**Issue by Leesoon1984** [tim-learn/DINE#2]
While Fine-tuning the distilled model, the performance on the  VisDA-C dataset drops, the results are as follows:

```
seed=2019: 74.98 --> 74.80
seed=2020: 76.17 --> 75.17
seed=2021: 79.8 --> 78.6.
```
The learning rate setting for VisDA-C dataset may matter.

**Comment by tim-learn** [comment]
> While Fine-tuning the distilled model, the performance on the VisDA-C dataset drops, the results are as follows:
> 
> ```
> seed=2019: 74.98 --> 74.80
> seed=2020: 76.17 --> 75.17
> seed=2021: 79.8 --> 78.6.
> ```
> 
> The learning rate setting for VisDA-C dataset may matter.

Sorry for the late reply, I will upload the code for VisDA-C in the following month.

**Comment by zou-yawen** [comment]
> While Fine-tuning the distilled model, the performance on the VisDA-C dataset drops, the results are as follows:
> 
> ```
> seed=2019: 74.98 --> 74.80
> seed=2020: 76.17 --> 75.17
> seed=2021: 79.8 --> 78.6.
> ```
> 
> The learning rate setting for VisDA-C dataset may matter.

hello, I try to repeat the result  for VisDA-C dataset with DINE,  I can't get the result could you please tell me your parameters
